src/test/groovy/edu/oregonstate/mist/inventory/AllowedValuesTest.groovy

```diff
@@ -0,0 +1,22 @@
+package edu.oregonstate.mist.inventory
+
+import org.junit.Test
+
+class AllowedValuesTest {
+
+    private AllowedValues testValues = new AllowedValues(list: ['foo', 'bar', 'eggplant'])
+
+    @Test
+    public void testPrettyString() {
+        String prettyString = testValues.pretty()
+        List<String> parsedPrettyString = prettyString.tokenize(',')
+
+        testValues.list.each {
```

Comment:
What about testing edge cases? What if there is only 0, 1 or 2 items in AllowedValues?